### ISSUE TITLE
Clear details on how to use!

### ISSUE THREAD

**Issue by vibezjit** [WorldDownloader/WorldDownloader#2]
The instructions on this make no sense and it gets confusing on where things go and how to start it up. Any help is appreciated. Thanks!

**Comment by Coosanta17** [comment]
Yes the instructions are so vague, I think I have an idea what they are trying to say but non-computer nerds like me (and the majority of users) need a clearer explanation. You just get the source code and somehow run it using mcpelauncher.
